lesson_09/homeworks.py

```diff
@@ -0,0 +1,75 @@
+#Функції для тестування
+
+#Функія №1 
+def list_of_strings(lst: list) -> list:
+    """Функція приймає список і повертає список тільки зі строкових значень"""
+    if lst == []:
+        raise ValueError("Список пустий!")
+    
+    if not isinstance(lst, list):
+        return "Введіть список!"
+    
+    lst_str = [elem for elem in lst if isinstance(elem, str)]
+
+    if not lst_str:
+        return "В списку немає строкових значень!"
```

Comment:
```suggestion
        raise ValueError("В списку немає строкових значень!")
```
І знову таки - раз ви заявили, що "Функція ... повертає список", то ви створюєте неочікувану для користувача ситуацію - він жеж прочитав докі і очікує список, а йому вертається... стрінга...